src/main/java/com/parasoft/findings/jenkins/coverage/api/metrics/steps/CoverageTableModel.java

```diff
@@ -95,13 +96,21 @@ public String getId() {
 
     @Override
     public TableConfiguration getTableConfiguration() {
-        TableConfiguration tableConfiguration = new TableConfiguration();
-        tableConfiguration.responsive();
+        CustomTableConfiguration customTableConfiguration = new CustomTableConfiguration();
+        customTableConfiguration.responsive();
         if (getId().contains("inline")) {
-            tableConfiguration.select(SelectStyle.SINGLE);
+            customTableConfiguration.select(SelectStyle.SINGLE);
+        }
+
+        customTableConfiguration.loadConfiguration();
+        String languageCode = LocaleProvider.getLocale().getLanguage();
+        String i18nFileBasename = null;
+        if ("zh".equals(languageCode)) {
```

Comment:
When browser language is set to **Chinese (Traditional)**, only the table header and footer shows in simplified Chinese, everything else is still English. We need to keep consistency in our plugin to show the table in English.